File: banda/JCZhang-Banda/JCZhang-Banda-JAVA/src/main/java/models/Pianist.java
package models;

import interfaces.IPianist;

public class Pianist extends Musician implements IPianist {

    String name;
    int yearsOfExperience;
    int numberOfKeys;
    static double salary = (Musician.salary * 1.3);
    public Pianist(String name, int yearsOfExperience, int numberOfKeys) {
        super(name, yearsOfExperience);
        this.name = name;
        this.yearsOfExperience = yearsOfExperience;
        this.numberOfKeys = numberOfKeys;

    }

    @Override
    public void playPiano() {
        System.out.println("im a pianist and im playing piano");
    }
}

File: banda/JCZhang-Banda/JCZhang-Banda_KOTLIN/src/main/kotlin/models/Pianist.kt
package models

import interfaces.IPianist

class Pianist(override var name: String?, override var yearsOfExperience: Int, var numberOfKeys: Int) :
    Musician(name, yearsOfExperience), IPianist {
    override fun playPiano() {
        println("im a pianist and im playing piano")
    }

    companion object {
        var salary = Musician.salary * 1.3
    }
}